java/dataflow-connector-examples/README.md

```diff
@@ -0,0 +1,77 @@
+# CBT / CDF examples
+
+A starter set of examples for writing Google Cloud Dataflow (CDF) programs using Cloud Bigtable (CBT).
+
+## HelloWorld - Writing Data
+
+The HelloWorld examples take two strings, converts them to their upper-case representation and writes
+them to Bigtable.
+
+This pipeline needs to be configured with four command line options for bigtable:
+
+ * --bigtableProject=[bigtable project]
+ * --bigtableClusterId=[bigtable cluster id]
+ * --bigtableZone=[bigtable zone]
+ * --bigtableTable=[bigtable tableName]
+
+HelloWorldParDo shows the use of ParDos - a simple user defined function - 
+https://cloud.google.com/dataflow/java-sdk/JavaDoc/com/google/cloud/dataflow/sdk/transforms/ParDo)
+to write to Bigtable.
+
+HelloWorldSink shows the use of a Sink - a construct that knows how to write Deletes and Puts.
+
+## SourceRowCount - Reading from CBT
+
+SourceRowCount shows the use of a Bigtable Source - a construct that knows how to scan a Bigtable Table.  SourceRowCount performs a simple row count using the Cloud Bigtable Source and writes the count to a file in Google Storage.
+
+## PubsubWordCount - Reading from Cloud Pubsub and writing to CBT
+
+The PubsubWordCount example reads from Cloud Pubsub and writes to CBT. It starts two jobs: one publishes messages to Cloud Pubsub, and the other one pulls messages, performs a word count for each message, and writes word count result to CBT. 
+
+## Running the examples
+
+To run the Hello world examples using managed resource in Google Cloud Platform, you should specify
+the following command-line options for dataflow:
+
+1. --project=<YOUR_PROJECT_ID>
+2. --stagingLocation=<STAGING_LOCATION_IN_CLOUD_STORAGE> 
+3. --runner=BlockingDataflowPipelineRunner
+
+Also, you need the following command line options for Bigtable:
+
+1. --bigtableProject=some_project 
+2. --bigtableClusterId=cluster_name 
+3. --bigtableZone=us-central1-b
+4. --bigtableTable=someTableName
+
+You can run this via maven, for example:
+
+`mvn exec:java -Dexec.mainClass="com.google.cloud.dataflow.sdk.io.bigtable.example.HelloWorldParDo" -Dexec.args="--runner=BlockingDataflowPipelineRunner --project=some_project --stagingLocation=gs://some_bucket --bigtableProject=some_project --bigtableClusterId=cluster_name--bigtableZone=us-central1-b --bigtableTable=someTableName"`
```

Comment:
a few things I'll fix -- parameter names mostly have Id at the end like bigtableZoneId, and there is a missing space between clusterID & zone